tools/proofers/for_mentors.php

```diff
@@ -273,10 +274,10 @@ function page_list_sql($mentored_round, $projectid)
         image
     ";
 
-    $page = mysqli_real_escape_string(DPDatabase::get_connection(), _("Page"));
-    $saved = mysqli_real_escape_string(DPDatabase::get_connection(), _("Saved"));
-    $proofreader = mysqli_real_escape_string(DPDatabase::get_connection(), _("Proofreader"));
-    $wc_events = mysqli_real_escape_string(DPDatabase::get_connection(), _("WordCheck Events"));
+    $page = DPDatabase::escape(_("Page"));
+    $saved = DPDatabase::escape(_("Saved"));
+    $proofreader = DPDatabase::escape(_("Proofreader"));
```

Comment:
do we need these as separate variables? I'd just sprintf below without creating these if possible.

Reply:
Technically no, but I think `sprintf` is a bit of a no-go here for exactly the same reason as above -- we've too many interpolated variables interleave with `DPDatabase::escape` values that we'd like to keep as interpolated variables.